external-docs/docs/guides/configuration/qube-config.md

```diff
@@ -121,56 +125,60 @@ There are several configuration options available across column types except obs
 | `definition_uri` | A uri of a resource to show how the column is created/managed (i.e. a uri of a PDF explaining a list of units) (Optional) | *none*                                                                           |
 
-The `from_existing` value when set provides the basis of linked data; it allows csvcubed to generate additional RDF-hints to allow users to discover how the `tidy_data.csv` links to other data semanticly.
+The `from_existing` value when set provides the basis of linked data; it allows csvcubed to generate additional RDF-hints to allow users to discover how the `tidy_data.csv` links to other data semantically.
 
-## Dimension configuration
+## Dimension Configuration
 
 | **field name**      | **description**                                                                                                                                                                                                      | **default value**                                                                |
 |---------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|----------------------------------------------------------------------------------|
 | `type`              | The type of the column (Required)                                                                                                                                                                                    | *dimension*                                                                      |
+| `from_template`     | Use a [column template](templates.md)                                                                                                                                                                                | *none*                                                                           |
 | `label`             | The title of the column (Optional)                                                                                                                                                                                   | The capital case of the header in the csv file with spaces replacing underscores |
 | `description`       | A description of the contents of the column (Optional)                                                                                                                                                               | *none*                                                                           |
 | `from_existing`     | The uri of the resource for reuse/extension (Optional)                                                                                                                                                               | *none*                                                                           |
```

Comment:
Similar to the csv/CSV notes, the term uri always appears in lower case in this documentation page, however in other parts of the documentation (e.g. the "Working with URIs" section) it is written in caps. I'm not sure which is best, as lower case is obviously suited for file extensions, but maybe caps is better for just normal references?

Reply:
Added to new ticket (#706)